src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRequestMessageExtensions.cs

```diff
@@ -34,7 +34,7 @@ internal static HttpRequestMessage Clone(this HttpRequestMessage request)
                 clone.Headers.TryAddWithoutValidation(header.Key, header.Value);
             }
 
-#if NET5_0
+#if NET5_0 || NET6_0
```

Comment:
According to https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#preprocessor-symbols .NET 6 should actually be setting NET5_0 as well.  This may have been a quirk of the particular SDK version we were using, I'm removed it and we can see if it builds.